src/analysis/types.md

```diff
@@ -42,21 +42,22 @@ from one platform to another, radare2 uses `definite` types like as
 `int8_t` or `uint64_t` and will convert `int` to `int32_t` or `int64_t`
 depending on the binary or debuggee platform/compiler.
 
-Basic types can be listed using `t` command, for the structured types
-you need to use `ts`, `tu` or `te` for enums:
+Basic types can be listed using `t` command. For the structured types
+you need to use `ts`, for unions use `tu` and for enums — `te`.
 
 ```
 [0x000051c0]> t
 char
 char *
+double
+float
 int
 int16_t
 int32_t
-int64_t
-int8_t
-long
-long long
-...
+[...]
```

Comment:
List them all. Update with the current output

Reply:
For some reason, I have gid_t, uid_t and pid_t out of the box. I'm not sure that's normal, but whatever.